Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM localhost:5000/tradingbase
```

Comment:
you will never have a local repo. Repo's will always be either on docker hub or in a private cloud registry such as ACS or google container registry. I recomend docker hub as it's free